### DISCUSSION TITLE
classes/class_webrtcpeerconnection

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#276] [User-contributed notes]
# classes/class_webrtcpeerconnection

Inherits: RefCounted< Object Inherited By: WebRTCPeerConnectionExtension Interface to a WebRTC peer connection. Description: A WebRTC connection between the local computer and a remote peer. Provid...

https://docs.godotengine.org/en/stable/classes/class_webrtcpeerconnection.html

**Comment by Bip901** [comment]
Note that WebRTCPeerConnection is *not* thread-safe. Calling any of its methods outside of the game's main thread fails with non-indicative messages like "Condition "interface == nullptr" is true. Returned: ERR_UNCONFIGURED"
